src/azimuth/gui/screen.c

```diff
@@ -181,6 +190,23 @@ void az_start_screen_redraw(void) {
   assert(display_initialized);
   glClear(GL_COLOR_BUFFER_BIT);
   glLoadIdentity();
+
+  // Calculate letterboxing scaling factor & offsets
+  SDL_GL_GetDrawableSize(window, &current_screen_width, &current_screen_height);
+  if( current_screen_width > current_screen_height ) {
```

Comment:
I think this conditional isn't quite right; if the window is square-ish, with the width just slightly larger than the height, then the sides of the image get cut off, like this:
![Screen Shot 2021-03-06 at 11 16 19 AM](https://user-images.githubusercontent.com/35096/110213508-5c998780-7e6e-11eb-9a2c-ce3bba6de4b4.png)
I think you can fix it by instead doing:
```c
  if (current_screen_width / (float)AZ_SCREEN_WIDTH >
      current_screen_height / (float)AZ_SCREEN_HEIGHT) {
```